src/Rings/ReesAlgebra.jl

```diff
@@ -0,0 +1,185 @@
+########################################################################
+# Rees algebras of modules 
+#
+# Following Eisenbud, Huneke, Ulrich: "What is the Rees algebra 
+# of a module?"; arXiv:math/0209187v1
+########################################################################
+
+@Markdown.doc """
+    rees_algebra(f::ModuleFPHom{<:ModuleFP, <:FreeMod}; check::Bool=true)
+
+For a *versal* (†) morphism ``f : M → F`` of a module ``M`` into a free 
+module ``F`` this computes the Rees algebra of ``M`` according to [1].
```

Comment:
Add the paper to the bib and use `[EHU03](@cite)` or so?

Is there a reason not to cite the published version, see <https://mathscinet.ams.org/mathscinet-getitem?mr=1937406> reps. <https://www.ams.org/journals/proc/2003-131-03/S0002-9939-02-06575-9/>?

Of course it may still be useful to give that arXiv link for readers w/o a subscription

Reply:
Citing the published version is even more accurate, because only the published version has the remark about the usage of the double dual for constructing a versal morphism. 

Thanks for the feedback with all the links below! I will look into that later today.